jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -84,39 +75,32 @@ jobs:
 ### Splitting Test Files
 {:.no_toc}
 
-The CLI supports splitting tests across machines
-when running parallel builds.
-To do this,
-pass a list of filenames to the `circleci tests split` command.
+The CLI supports splitting tests across machines when running parallel jobs.
 
-    circleci tests split test_filenames.txt
+![Test Splitting]({{ site.baseurl }}/assets/img/docs/test_splitting.png)
 
-The CLI looks up the number of containers,
-along with the current container index.
-Then, it uses deterministic splitting algorithms
-to split the test files across all available containers.
+#### Splitting by Timing Data
 
-By default,
-the number of containers is specified by the `parallelism` key.
-You can manually set this
-by using the `--total` flag.
+The best way to optimize your test suite across a set of parallel executors is to split your tests using timing data. This will ensure the tests are split in the most even way, leading to a shorter overall test time.
 
-    circleci tests split --total=4 test_filenames.txt
+On each successful run of a test suite, CircleCI saves timings data to a directory specified by the path in the [`store_test_results`]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results) step. This timings data consists of how long each test took to complete per filename or classname, depending on the language you are using.
```

Comment:
It might be better to say that we save the timing data _from_ the directory specified by `store_test_results`. We take that timing data and process it to make it available in later builds.